sass/body.sass

```
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700,800&display=swap')
+
+
+body
+  background-color: $background-color
+  font-family: "Open Sans", sans-serif
+
+.top-container
+  background-image: $backgroundImageMobile
+  background-repeat: no-repeat no-repeat
+  background-size: 100vw
+  @media (min-width: $min-width)
+    background-image: linear-gradient(to bottom,  rgba($white,0) 0%, rgba($white,0) 50%, $light50 80%, $light 100%), $backgroundImage
```

Why did you use `rgba($white,0)`? I'm asking because in this case it's a better solution than using `transparent`, because Safari has problems with such gradients. So I'm wondering if it's deliberate or by accident :)

Could be made a little shorter by omitting the default direction and the first color.

```suggestion
    background-image: linear-gradient(rgba($white,0) 50%, $light50 80%, $light 100%), $backgroundImage
```